packages/main/src/features/InputElementsFormSupport.ts

```diff
@@ -64,13 +60,13 @@ class FormSupport {
 	 * @param { NativeInputUpdateCallback } nativeInputUpdateCallback - callback to calculate the native input's "disabled" and "value" properties
 	 * @param { NativeInputChangeCallback } nativeInputChangeCallback - callback, added to native input's "change" event
 	 */
-	static syncNativeFileInput(element: IFormFileElement, nativeInputUpdateCallback: NativeInputUpdateCallback, nativeInputChangeCallback: NativeInputChangeCallback) {
+	static syncNativeFileInput(element: IFormElement, nativeInputUpdateCallback: NativeInputUpdateCallback, nativeInputChangeCallback: NativeInputChangeCallback) {
 		const needsNativeInput = !!element.name;
-		let nativeInput = element.querySelector(`input[type=${element._type || "hidden"}][data-ui5-form-support]`) as HTMLInputElement;
+		let nativeInput = element.querySelector(`input[type="file"][data-ui5-form-support]`) as HTMLInputElement;
```

Comment:
@dimovpetar get _type() from FileUploader is now removed and "file" directly used in "InputElementsFormSuport#syncNativeFileInput"

Reply:
Looks good